API/Collection/ObjectStateCollection.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Kaliop\eZMigrationBundle\API\Collection;
+
+use eZ\Publish\API\Repository\Values\ObjectState\ObjectState;
+
+/**
+ * @todo add phpdoc to suggest typehinting
+ */
+class ObjectStateCollection extends AbstractCollection
+{
+    protected $allowedClass = ObjectState::class;
```

Comment:
Cool, I never thought about his syntax. Is it compatible all the way back to old pp versions ?